milli/src/update/index_documents/extract/extract_docid_word_positions.rs

```diff
@@ -183,3 +257,36 @@ fn process_tokens<'a>(
         })
         .filter(|(_, t)| t.is_word())
 }
+
+fn potential_language_detection_error(languages_frequency: &Vec<(Language, usize)>) -> bool {
```

Comment:
```suggestion
fn potential_language_detection_error(languages_frequency: &[(Language, usize)]) -> bool {
```

I'm a bit surprised this is not flagged by clippy

Reply:
Because the function is used in an iterator:
```rust
if script_language_word_count.values().any(potential_language_detection_error) {
```

This would force me to change this implementation instead of directly passing the function. You may have a tip about this.

Reply:
[Here's a playground with various options](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=cc5e25c75985c768dc63ec9d68b811e2)

Reply:
I think that the solution with `map(Vec::as_slice)` is the more suitable, the others are way less readable for an idealistic way of coding 😬

Reply:
That's a personal style question, I personally don't have anything against lambdas 😅

Reply:
Well, I like lambdas, but sometimes, being able to read your code as a sentence is enjoyable, like:
`if in script_language_word_count values there is any potential_language_detection_error then do ...` 😋 😋